packages/superset-ui-legacy-preset-chart-nvd3/src/utils.js

```diff
@@ -58,29 +60,31 @@ export function drawBarValues(svg, data, stacked, axisFormat) {
         })
       : [];
 
-  const groupLabels = svg.select('g.nv-barsWrap').append('g');
-
-  svg
-    .selectAll('g.nv-group')
-    .filter((d, i) => !stacked || i === countSeriesDisplayed - 1)
-    .selectAll('rect')
-    .each(function each(d, index) {
-      const rectObj = d3.select(this);
-      if (rectObj.attr('class').includes('positive')) {
-        const transformAttr = rectObj.attr('transform');
-        const yPos = parseFloat(rectObj.attr('y'));
-        const xPos = parseFloat(rectObj.attr('x'));
-        const rectWidth = parseFloat(rectObj.attr('width'));
-        const textEls = groupLabels
-          .append('text')
-          .attr('y', yPos - 5)
-          .text(format(stacked ? totalStackedValues[index] : d.y))
-          .attr('transform', transformAttr)
-          .attr('class', 'bar-chart-label');
-        const labelWidth = textEls.node().getBBox().width;
-        textEls.attr('x', xPos + rectWidth / 2 - labelWidth / 2); // fine tune
-      }
-    });
+  svg.selectAll('.bar-chart-label').remove();
+  setTimeout(() => {
+    const groupLabels = svg.select('g.nv-barsWrap').append('g');
```

Comment:
This will create new `g` every time. Is that expected?

Reply:
Ah, good point. I'll fix this.